registry.yaml

```
@@ -432,3 +432,22 @@ plugins:
         source: k8s_audit
       extraction:
         supported: true
+  - name: falco_journald_plugin
+    description: Read Journald events into Falco
+    authors: Grzegorz Nosek
+    contact: https://github.com/gnosek/falco-journald-plugin
+    maintainers:
+      - name: Grzegorz Nosek
+    url: https://github.com/gnosek/falco-journald-plugin
+    license: Apache-2.0
+    keywords:
+      - syslog
+      - systemd
+      - journald
+    capabilities:
+      sourcing:
+        supported: true
+        id: 514
```

```suggestion
        id: 17
```

Using the port number was nice, but unfortunately :sweat:  :point_down: 

https://github.com/falcosecurity/plugins/blob/main/docs/plugin-ids.md#assigning-an-id
>When making your request, please choose the next available ID in the [registry.yaml](https://github.com/falcosecurity/plugins/blob/main/registry.yaml) file. The id will be definitively assigned to your plugin once the corresponding PR is merged, and the [registry.yaml](https://github.com/falcosecurity/plugins/blob/main/registry.yaml) file is updated.

---

:sadpanda: will update ;)